tmcprototype/DishLeafNode/DishLeafNode/DishLeafNode.py

```diff
@@ -210,7 +213,81 @@ def commandCallback(self, event):
             print(CONST.ERR_EXCEPT_CMD_CB, except_occurred)
             self._read_activity_message = CONST.ERR_EXCEPT_CMD_CB + str(except_occurred)
             self.dev_logging(CONST.ERR_EXCEPT_CMD_CB, int(tango.LogLevel.LOG_ERROR))
-    # PROTECTED REGION END #    //  DishLeafNode.class_variable
+
+
+    def convert_radec_to_azel(self, data):
+        try:
+            print ("data is:", data)
+            print(type(data))
+            # Setting Observer Position as Pune
+            a = katpoint.Antenna(name='d1', latitude='18:31:48:00', longitude='73:50:23.99', altitude=570)
```

Comment:
Use of variable names that makes sense.

Reply:
Done.

Reply:
Commit id is: [01a44358029a01fcc943c419fc4d7606eab518be](https://github.com/ska-telescope/tmc-prototype/commit/01a44358029a01fcc943c419fc4d7606eab518be)